packages/reqresp/src/metrics.ts

```diff
@@ -15,6 +15,16 @@ export function getMetrics(register: MetricsRegisterExtra) {
       help: "Counts total requests done per method",
       labelNames: ["method"],
     }),
+    outgoingOpennedStreams: register.gauge<{method: string}>({
+      name: "beacon_reqresp_outgoing_opened_streams_total",
+      help: "Counts total opened streams per method",
+      labelNames: ["method"],
+    }),
+    outgoingClosedStreams: register.gauge<{method: string}>({
```

Comment:
would be good if we make this a counter
```suggestion
    outgoingClosedStreams: register.counter<{method: string}>({
```